src/core/textrenderer/qgsfontmanager.cpp

```diff
@@ -192,1687 +221,1684 @@ void QgsFontManager::enableFontDownloadsForSession()
-    QStringLiteral( "Vina Sans" ),
-    QStringLiteral( "Voces" ),
-    QStringLiteral( "Volkhov" ),
-    QStringLiteral( "Vollkorn" ),
-    QStringLiteral( "Vollkorn SC" ),
-    QStringLiteral( "Voltaire" ),
-    QStringLiteral( "Vujahday Script" ),
-    QStringLiteral( "Waiting for the Sunrise" ),
-    QStringLiteral( "Wallpoet" ),
-    QStringLiteral( "Walter Turncoat" ),
-    QStringLiteral( "Warnes" ),
-    QStringLiteral( "Water Brush" ),
-    QStringLiteral( "Waterfall" ),
-    QStringLiteral( "Wavefont" ),
-    QStringLiteral( "Wellfleet" ),
-    QStringLiteral( "Wendy One" ),
-    QStringLiteral( "Whisper" ),
-    QStringLiteral( "WindSong" ),
-    QStringLiteral( "Wire One" ),
-    QStringLiteral( "Wix Madefor Display" ),
-    QStringLiteral( "Wix Madefor Text" ),
-    QStringLiteral( "Work Sans" ),
-    QStringLiteral( "Xanh Mono" ),
-    QStringLiteral( "Yaldevi" ),
-    QStringLiteral( "Yanone Kaffeesatz" ),
-    QStringLiteral( "Yantramanav" ),
-    QStringLiteral( "Yatra One" ),
-    QStringLiteral( "Yellowtail" ),
-    QStringLiteral( "Yeon Sung" ),
-    QStringLiteral( "Yeseva One" ),
-    QStringLiteral( "Yesteryear" ),
-    QStringLiteral( "Yomogi" ),
-    QStringLiteral( "Yrsa" ),
-    QStringLiteral( "Ysabeau" ),
-    QStringLiteral( "Ysabeau Infant" ),
-    QStringLiteral( "Ysabeau Office" ),
-    QStringLiteral( "Ysabeau SC" ),
-    QStringLiteral( "Yuji Boku" ),
-    QStringLiteral( "Yuji Hentaigana Akari" ),
-    QStringLiteral( "Yuji Hentaigana Akebono" ),
-    QStringLiteral( "Yuji Mai" ),
-    QStringLiteral( "Yuji Syuku" ),
-    QStringLiteral( "Yusei Magic" ),
-    QStringLiteral( "ZCOOL KuaiLe" ),
-    QStringLiteral( "ZCOOL QingKe HuangYou" ),
-    QStringLiteral( "ZCOOL XiaoWei" ),
-    QStringLiteral( "Zen Antique" ),
-    QStringLiteral( "Zen Antique Soft" ),
-    QStringLiteral( "Zen Dots" ),
-    QStringLiteral( "Zen Kaku Gothic Antique" ),
-    QStringLiteral( "Zen Kaku Gothic New" ),
-    QStringLiteral( "Zen Kurenaido" ),
-    QStringLiteral( "Zen Loop" ),
-    QStringLiteral( "Zen Maru Gothic" ),
-    QStringLiteral( "Zen Old Mincho" ),
-    QStringLiteral( "Zen Tokyo Zoo" ),
-    QStringLiteral( "Zeyada" ),
-    QStringLiteral( "Zhi Mang Xing" ),
-    QStringLiteral( "Zilla Slab" ),
-    QStringLiteral( "Zilla Slab Highlight" ),
-  };
+    fontUrls.append( QStringLiteral( "https://github.com/google/fonts/raw/main/%1" ).arg( path ) );
+  }
+  return QgsFontDownloadDetails(
+           family,
+           fontUrls,
+           !licensePath.isEmpty() ? QStringLiteral( "https://github.com/google/fonts/raw/main/%1" ).arg( licensePath ) : QString()
+         );
+}
 
-// temporarily unavailable -- see https://github.com/google/fonts/issues/7481
-#if 0
-  auto cleanFontFamily = []( const QString & family ) -> QString
+QgsFontDownloadDetails QgsFontManager::detailsForFontDownload( const QString &family, QString &matchedFamily ) const
+{
+  const thread_local std::vector< QgsFontDownloadDetails > sGoogleFonts
```

Comment:
A code comment indicating how this list has been generated could be useful. I suspect some script might have been used?

Reply:
@rouault ok, I've just checked that in now